tasks/custom_task/custom_task.py

```diff
@@ -15,6 +15,11 @@
 from invoke import Context
 
 DD_INVOKE_LOGS_PATH = "/tmp/dd_invoke.log"
+if sys.platform == 'win32':
+    try:
+        DD_INVOKE_LOGS_PATH = f"{os.environ['TEMP']}\\dd_invoke.log"
+    except Exception as e:
+        print(f"Warning: couldn't set the DD_INVOKE_LOGS_PATH (error: {e})", file=sys.stderr)
```

Comment:
What happens if an exception is thrown in this case? Will the `custom__call__` fail with an error message each time a Windows user runs an invoke task?

If that's the case, you might need to write a dummy `custom__call__` function with a noop so you won't spam them

Reply:
Fixed in [bb5378b](https://github.com/DataDog/datadog-agent/pull/25540/commits/bb5378bf8881ac9e56d4a996353748c7b08f6e47) ! WDYT?